src/components/Certify/Certify.vue

```diff
@@ -0,0 +1,157 @@
+<template>
+  <v-card flat id="AR-step-4-container">
+    <div class="certify-container">
+      <div class="certifiedby-container">
+        <label>Legal Name</label>
+        <div class="value certifiedby">
+          <v-text-field
+            filled
+            persistent-hint
+            id="certified-by-textfield"
+            label="Person's legal name"
+            hint="Legal name of authorized person"
+            :value="certifiedBy"
+            :rules="[(v) => !!v || 'A person\'s legal name is required.']"
+            @input="emitCertifiedBy($event)"
+          />
+        </div>
+      </div>
+      <v-checkbox
+        :value="isCertified"
+        @change="emitIsCertified($event)"
+        id="isCertified-checkbox"
+      >
+        <template slot="label">
+          <div class="certify-stmt">
+            I,
+            <strong>{{ trimmedCertifiedBy || "[Legal Name]" }}</strong>
+            , certify that I have relevant knowledge of the
+            {{ entityDisplay || "association" }} and that I am authorized to
+            make this filing.
+          </div>
+        </template>
+      </v-checkbox>
+      <p class="certify-clause">Date: {{ currentDate }}</p>
+      <p class="certify-clause">{{ message }}</p>
```

Comment:
Does business-edit-ui have a requirement to display email addresses here?

https://github.com/bcgov/bcrs-business-create-ui/blob/928c46e881008572ac60893ce7a3eec34d48c210/src/components/ReviewConfirm/Certify.vue#L39